api/desecapi/tests/test_rrsets.py

```diff
@@ -199,16 +199,16 @@ def test_create_other_rr_sets(self):
 
     def test_create_my_rr_sets_too_long_content(self):
         def _create_data(length):
-            content_string = 'A' * (length - 2)  # we have two quotes
-            return {'records': [f'"{content_string}"'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{length}'}
+            content_string = 'A' * length
+            return {'records': [f'{content_string}'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{length}'}
 
         with self.assertPdnsRequests(self.requests_desec_rr_sets_update(self.my_empty_domain.name)):
-            response = self.client.post_rr_set(self.my_empty_domain.name, **_create_data(500))
+            response = self.client.post_rr_set(self.my_empty_domain.name, **_create_data(255))
```

Comment:
We should allow longer record contents and somehow work around the chunking issue

Reply:
done